05-UI-ReactJS/topic-3/src/App.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Movies from './components/Movies';
+import AddMovie from './components/AddMovie';
+//import PropTypes from 'prop-types';
+
+import './App.css';
+
+class App extends React.Component {
+  constructor() {//defining initial keys
```

Comment:
Have in mind when you set a constructor for an extended component you must use the props stuff. Take a look at this